draft-packages/form/KaizenDraft/Form/Primitives/Label/styles.scss

```diff
@@ -41,9 +41,13 @@ $label-start-margin: 10px;
 
   display: flex;
 
-  .labelText {
+  .prependedLabel {
+    @include ca-margin($end: $label-start-margin);
+    order: -1; // place label before the control
+  }
+
+  .appendedLabel {
```

Comment:
Perhaps we should just bring these styles out of this nesting, and that way we should only need to have it written once.

If that doesn't work, I don't think there's any way to end up with a `prependedLabel` with a checkbox or radio, so we could delete that class from here.